optimum/exporters/onnx/config.py

```diff
@@ -341,6 +341,34 @@ def __init__(
 
             self.DUMMY_INPUT_GENERATOR_CLASSES += self._past_key_values_generator
 
+    @property
+    def inputs(self) -> Dict[str, Dict[int, str]]:
+        common_inputs = {}
+        if self._behavior is not ConfigBehavior.DECODER:
+            common_inputs["input_ids"] = {0: "batch_size", 1: "encoder_sequence_length"}
+
+        common_inputs["attention_mask"] = {0: "batch_size", 1: "encoder_sequence_length"}
+
+        if self._behavior is not ConfigBehavior.ENCODER:
+            # TODO: it is likely this pop() is unwanted as we then always hit
+            # https://github.com/huggingface/transformers/blob/v4.26.0/src/transformers/models/t5/modeling_t5.py#L965-L969
+            common_inputs.pop("attention_mask")
+
+            if self.use_past_in_inputs:
+                # TODO: validate the axis name for attention_mask
+                # common_inputs["attention_mask"][1] = "past_encoder_sequence_length + sequence_length"
```

Comment:
?

Reply:
This was copied from `class TextSeq2SeqOnnxConfig(OnnxSeq2SeqConfigWithPast):` so if in future change is made it is done in both places.[L167](https://github.com/huggingface/optimum/blob/main/optimum/exporters/onnx/config.py#L167)